docs/tutorial-run-solver.md

```diff
@@ -0,0 +1,176 @@
+---
+id: tutorial-solver
+title: Quickstart - Running your own Solver
+sidebar_label: Quickstart
+---
+
+#
+
+# Contents
+
+1. [Installation](#installing-driver-tools)
+2. [Building The Driver](#building-the-driver)
+3. [Configuring Environment](#configuring-environment-variables)
+4. [Acquiring the Orderbook](#acquiring-the-orderbook)
+5. [Run the Solver](#run-the-solver)
+6. [Advanced Configuration](#advanced-configuration)
+   1. [Orderbook Filter](#Configuring-an-Orderbook-Filter)
+
+---
+
+## Installing Driver Tools
+
+Clone the dex-services repository, and acquire the project requirements
+
+```sh
+git clone git@github.com:gnosis/dex-services.git
+```
+
+We demonstrate how to run the open-solver. Once comfortable, the open-solver can be adapted to suit your own needs from the open-solver repo
+
+```sh
+git clone git@github.com:gnosis/dex-solver.git
+```
+
+## Building The Driver
+
+Build the driver (with open-solver dockerhub image) from within the root directory of `dex-services` repo. This command builds the driver while simultanesouly baking the open solver into the the build.
+
+```sh
+docker-compose build --build-arg SOLVER_BASE=gnosispm/dex-open-solver:master stablex-debug
+```
+
+Note that, if `SOLVER_BASE` is not specified, the driver's own internal `naive-solver` will be used. Alternatively, if you have your own personalized solver, this image would need to be specified during the build.
+We will see that specifying solver type again at runtime is also expected and this may seem counterintuitive.
```

Comment:
I think for now I will just remove this since moving it will require substantial rewording. Will check to see if this is still necessary as I parse through the latter sections.